labman/gui/handlers/auth.py

```diff
@@ -47,3 +51,24 @@ class LogoutHandler(BaseHandler):
     def get(self):
         self.clear_cookie("user")
         self.redirect("/")
+
+
+class AccessHandler(BaseHandler):
+    @authenticated
+    def get(self):
+        self.render('access.html', users=User.list_users(),
+                    access_users=User.list_users(access_only=True))
```

Comment:
Does this only show users that have access? Is it possible to grant access to a user that isn't already on the list?